uiowa/uiowa
# Tippie pardot2

Resolves #3811

Assumes "page" confirmation method is set.

# To Test
- Customer is checking the integration with Pardot.
- Technical review of the code.
- Sync tippie down and as a webmaster ensure that you can add the handler without error and have settings saved (included/excluded values)
- Create a hidden field "visitorid" on a webform. See if that field has a submission value.

# ToDo

- [x] Customer approval from testing in DEV
- [x] Uncomment environment restriction [docroot/modules/custom/uiowa_core/uiowa_core.module]
